daemon/daemon.go

```diff
@@ -105,8 +107,12 @@ func (d *daemon) LoadConfig(newConfig config.Config) error {
 		return fmt.Errorf("unrecognized handler %s", d.proxy.Handler)
 	}
 
+	middleware.EnableRollups(context.Background(), logger.New("sphinx"), 20*time.Second)
 	d.handler = middleware.New(handler, "sphinx", func(req *http.Request) map[string]interface{} {
-		return map[string]interface{}{"guid": req.Header.Get("X-Request-Id")}
+		return map[string]interface{}{
+			"guid": req.Header.Get("X-Request-Id"),
```

Comment:
So `guid` is going to get clobbered in roll-ups, but will still show up in errors (since they aren't rolled up) and any logs in the request that use `logger.FromContext(ctx)`?

Reply:
:nods: